pkg/apis/submariner.io/v1/types.go

```diff
@@ -77,10 +79,20 @@ type GatewayStatus struct {
 	Version       string       `json:"version"`
 	HAStatus      HAStatus     `json:"haStatus"`
 	LocalEndpoint EndpointSpec `json:"localEndpoint"`
+	// +optional
+	Latency       LatencySpec  `json:"latency"`
```

Comment:
Since Latency is for each connection, this should be part of "Connections" struct.

Reply:
Done